DotNetDBF/DBFWriter.cs

```diff
@@ -145,6 +146,16 @@ public byte LanguageDriver
             }
         }
 
+	public string NullSymbol
+        {
+            get { return _nullSymbol; }

```

Comment:
I'd prefer `?? DBFFieldType.Unknown`. That way the `NullSymbol` will return the default when `null`.